### DISCUSSION TITLE
Lodestar Standup 2024-42 (November 26, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#7219] [Standups]
### Meeting Info
[November 26, 2024, 15:00-16:00 UTC](https://savvytime.com/converter/utc-to-pst-est-cet-ist-ict/nov-08-2024/3pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 15:00 UTC

### Agenda
- Open Discussion
  - Post-Devcon updates
- Team Updates
- Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by philknows** [comment]
Our team should come to consensus on whether or not we should include the fee market with 7742. See: https://github.com/ethereum/EIPs/pull/9047

**Comment by twoeths** [comment]
when investigating nft-devnet, I spin up a lodestar node in our infrastructure and feel very comfortable with that. So I suggest we do that for all devnets with the benefit:
- it's very accessible and transparent for the whole team
- if we have a fix for a PR, we can deploy to that node and verify
- it's easier to investigate the issue: we get the log we want, can connect to debug etc. We cannot do that in EF's dev nodes

if we agree on that we should:
- how many devnets do we join, how many nodes we want per devnet and type of node, is our infrastructure enough?
- need help from Infrastructure team to write scripts to deploy a node to a devnet in seconds (both CL + EL) given devnet's config

**Reply by twoeths** [reply]
@philknows created this https://github.com/ChainSafe/lodestar-quickstart/pull/58

**Comment by philknows** [comment]
We should discuss this a little bit more: https://github.com/ChainSafe/lodestar/issues/7237

and then to clarify intentions of PRs to support Deno such as https://github.com/ChainSafe/ssz/pull/423#issuecomment-2497565182

**Comment by philknows** [comment]
Should also get an update on where we stand with PeerDAS devnets and fork following. 
They're looking to launch peerdas-devnet-4 this week if we haven't rebased it off Electra yet. Or mid-December target which includes rebasing off Electra.

**Comment by philknows** [comment]
Closing for #7263. Notes can be found at https://github.com/ChainSafe/lodestar/wiki/Lodestar-Planning-&-Standup-Meetings#november-26-2024-planning-and-standup-meeting
